schema-registry/core/src/main/java/com/hortonworks/registries/schemaregistry/SchemaMetadataStorable.java

```diff
@@ -224,8 +230,17 @@ public static SchemaMetadataStorable updateSchemaMetadata(SchemaMetadataStorable
         schemaMetadataStorable.setSchemaGroup(schemaMetadata.getSchemaGroup());
         schemaMetadataStorable.setName(schemaMetadata.getName());
         schemaMetadataStorable.setDescription(schemaMetadata.getDescription());
-        schemaMetadataStorable.setCompatibility(schemaMetadata.getCompatibility());
-        schemaMetadataStorable.setValidationLevel(schemaMetadata.getValidationLevel());
+
+        schemaMetadataStorable.setCompatibility(
```

Comment:
the if null check / defaulting is done already in schema meta data. 

https://github.com/hortonworks/registry/blob/master/schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/SchemaMetadata.java